### PR TITLE
[14.0][ADD] github_connector_branch_options

### PR DESCRIPTION
**Pull request by Volodiay622**

### CODE REVIEW

**Comment by legalsylvain** [comment]
Same remarks as in #101 

Hi @Volodiay622
Thanks for your contribution !
That's a great addition.

however I don't see the need to create a new module. (this module does not add any additional dependency + adds an optional useful feature).
I would be more in favor of making a modification to the existing github_connector module.

What do you think about it ?

**Comment by ivs-cetmix** [comment]
> Same remarks as in #101
> 
> Hi @Volodiay622 Thanks for your contribution ! That's a great addition.
> 
> however I don't see the need to create a new module. (this module does not add any additional dependency + adds an optional useful feature). I would be more in favor of making a modification to the existing github_connector module.
> 
> What do you think about it ?

I think this is a good idea. We will open a new PR for that. Meanwhile could you please do a brief review just to ensure we don't have any major issues with the code?

**Comment by legalsylvain** [comment]
> Meanwhile could you please do a brief review just to ensure we don't have any major issues with the code?

Very quick overview. Lgtm. But I dont use this module anymore. I just developed it some years ago for the OCA.

**Comment by ivs-cetmix** [comment]
> > Meanwhile could you please do a brief review just to ensure we don't have any major issues with the code?
> 
> Very quick overview. Lgtm. But I dont use this module anymore. I just developed it some years ago for the OCA.

Ok thank you! Meanwhile just curious: what are you using instead now?

**Comment by Volodiay622** [comment]
> Very quick overview. Lgtm. But I dont use this module anymore. I just developed it some years ago for the OCA.

Hi, thanks for the feedback!

**Comment by legalsylvain** [comment]
> what are you using instead now?

Nothing !

File: github_connector_branch_options/models/github_repository_branch.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    "7",
+    "8",
+    "9",
+    "10",
+    "11",
+    "12",
+    "13",
+    "14",
+    "15",
+    "16",
```
Comment (legalsylvain) [inline github_connector_branch_options/models/github_repository_branch.py:32]: don't understand that values.

File: github_connector_branch_options/models/github_repository.py [hunk 2]
```diff
@@ -7,25 +7,57 @@
 
 _logger = logging.getLogger(__name__)
 
+_odoo_version = [
```
Comment (legalsylvain) [inline github_connector_branch_options/models/github_repository.py:10]: github_connector is agnostic. if you want to develop "odoo" things, please put in github_connector_odoo module.

File: github_connector_branch_options/models/github_repository_branch.py (hunk 1 above)
Comment (Volodiay622) [reply github_connector_branch_options/models/github_repository_branch.py:32]: if the odoo version is specified in the branch name in this format (not 12.0-my_branch-... but 12-my_branch-...)

**Comment by Volodiay622** [comment]
will be reopened for the github_connector_odoo module

**Pull request closed without merging**